src/Faker/Name.cs

```diff
@@ -46,6 +46,15 @@ public static class Name
 		/// <returns>The randomly created first name.</returns>
 		public static string First()
 		{
+			return Resources.Name.First.Split(Config.SEPARATOR).Random().Trim();
+		}
+
+		/// <summary>
+		///   Creates a random first name, using a more aggressive caching strategy.
+		/// </summary>
+		/// <returns>The randomly created first name.</returns>
+		public static string FirstCached()
```

Comment:
I'll need to think a little about it.
Mainly the reasoning I have about it being an overload with a boolean value is the following.
1. Caching have the potential of breaking stuff, which would make this a breaking change.
2. Sometime we would like to force retrieval of a non cached value (although highly unlikely)